### PR TITLE
[DEVOPS-188]  Assign *.aws.iohkdev.io to report-server and explorer

### PR DESCRIPTION
**Pull request by deepfire**
Functionally complete -- remains to be ported to `staging` and `production`.

Testing scenario: https://issues.serokell.io/issue/DEVOPS-188#comment=96-5254

cc @domenkozar

### CODE REVIEW

**Comment by deepfire** [comment]
@domenkozar, is this alternative approach I just implemented good, or should I go ahead with new files?

Sorry, I only saw your design comment after I pushed the actual implementation..

**Comment by domenkozar** [comment]
```
{ pkgs, config, ...}: {
  imports = [ ./../modules/cardano-node-prod.nix ];

  deployment.ec2....
}
```

File: deployments/cardano-explorer-env-development.nix [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 with (import ./../lib.nix);
 {
-  sl-explorer = { pkgs, ...}: {
+  sl-explorer = { pkgs, config, ...}: {
```
Comment (domenkozar) [inline deployments/cardano-explorer-env-development.nix]: cleanup

**Comment by domenkozar** [comment]
Pushed f3868d0cbae50ecc6ea6e7bf2d7ae2295290a67b to staging, thanks!

**Pull request closed without merging**